UILabel_Typography_Extensions/Typography/UILabel+Observer.swift

```diff
@@ -44,19 +44,15 @@ extension UILabel {
 		}
 	}
 	
-	func observeIfNeeded() {
+    func observeTextChange(onChange completion: @escaping ()->()) {
```

Comment:
Thanks, makes a lot of sense to move out the use-case from the extension, leaving it concerned with observing only. 🙌